src/HealthGPS/risk_factor_adjustable_model.cpp

```diff
@@ -193,15 +200,20 @@ RiskFactorAdjustableModel::calculate_simulated_mean(Population &population,
 
 RiskFactorAdjustableModelDefinition::RiskFactorAdjustableModelDefinition(
     std::unique_ptr<RiskFactorSexAgeTable> expected,
-    std::unique_ptr<std::unordered_map<core::Identifier, double>> expected_trend)
-    : expected_{std::move(expected)}, expected_trend_{std::move(expected_trend)} {
+    std::unique_ptr<std::unordered_map<core::Identifier, double>> expected_trend,
+    std::unique_ptr<std::unordered_map<core::Identifier, int>> trend_steps)
```

Comment:
Am I right in thinking that `expected_trend` and `trend_steps` will have the same keys? If so, would it make sense to just have the one map, with each value being a struct containing both of these values?

Reply:
Mind If I do this one later? Still not completely certain that a new bunch of vars won't be requested soon.